### PR TITLE
refactor: adjust dimension of compact transaction arrow

### PR DESCRIPTION
**Pull request by dated**
## Summary

https://app.clickup.com/t/rga0t7
https://app.clickup.com/t/rar9xe

## Checklist

-   [ ] My changes look good in both light AND dark mode
-   [ ] The change is not hardcoded to a single network, but has multi-asset in mind
-   [ ] I checked my changes for obvious issues, debug statements and commented code
-   [ ] Documentation _(if necessary)_
-   [x] Tests _(if necessary)_
-   [x] Ready to be merged

### CODE REVIEW

**Review by samharperpittam — APPROVED** [review]

**Comment by samharperpittam** [comment]
@dated - It looks like with this avatar change, the 'return' icon clashes with the avatar icon slightly.
image

File: src/domains/transaction/components/TransactionTable/TransactionRow/TransactionRowRecipientIcon.tsx [hunk 1]
```diff
@@ -15,11 +15,24 @@ interface Properties {
...
+	if (isCompact) {
+		return (
+			<span
+				data-testid="TransactionRowRecipientIcon"
+				className="w-5 h-5 flex items-center justify-center text-theme-text dark:text-theme-secondary-600"
+			>
+				<Icon name={getIcon(type)} size="lg" />
+			</span>
+		);
 	}
```
Comment (w3ea) [inline src/domains/transaction/components/TransactionTable/TransactionRow/TransactionRowRecipientIcon.tsx:37]: i think there is no need for this
just make border color transparent ([here](https://github.com/PayvoHQ/wallet/blob/e6d0efee09815cb74596a174298470865df7fd84/src/domains/transaction/components/TransactionTable/TransactionRow/TransactionRowRecipientIcon.tsx#L42-L45))
```
className={cn(
    "text-theme-text dark:text-theme-secondary-600",
    {"border-theme-text dark:border-theme-secondary-600": !isCompact},
    {"border-transparent": isCompact},
    shadowClasses
)}
```

and of course adding xs size [in circle.styles](https://github.com/PayvoHQ/wallet/blob/e6d0efee09815cb74596a174298470865df7fd84/src/app/components/Circle/Circle.styles.ts#L10)

Comment (dated) [reply src/domains/transaction/components/TransactionTable/TransactionRow/TransactionRowRecipientIcon.tsx:37]: it's cleaner without the transparent circle imho.

Comment (w3ea) [reply src/domains/transaction/components/TransactionTable/TransactionRow/TransactionRowRecipientIcon.tsx:37]: except border, these lines ([here](https://github.com/PayvoHQ/wallet/blob/e6d0efee09815cb74596a174298470865df7fd84/src/domains/transaction/components/TransactionTable/TransactionRow/TransactionRowRecipientIcon.tsx#L29-L34), [here](https://github.com/PayvoHQ/wallet/blob/e6d0efee09815cb74596a174298470865df7fd84/src/domains/transaction/components/TransactionTable/TransactionRow/TransactionRowRecipientIcon.tsx#L39-L48)) are same imo :idk:

Comment (dated) [reply src/domains/transaction/components/TransactionTable/TransactionRow/TransactionRowRecipientIcon.tsx:37]: but we avoid computing and polluting the dom with the styled component.

Comment (w3ea) [reply src/domains/transaction/components/TransactionTable/TransactionRow/TransactionRowRecipientIcon.tsx:37]: then we should fix these codes too
[here](https://github.com/PayvoHQ/wallet/commit/417110180a5e3ac872df808dedb4df9c648c392a#diff-b6458201759c05307b9a281baf86c974bc266d415a6b6d61cc105fa1b5d926b4R62), [here](https://github.com/PayvoHQ/wallet/commit/f3f543d3b3e6a38f71923881f25e0e8a417caa9e#diff-6ee4d4f5f4fbcb8fc7137c3859a8a6f97adeabf0a5991f43b28ae96208f5559dR21), [here](https://github.com/PayvoHQ/wallet/commit/d8858844f0f1b544bf2dad8bc8ca7ab41bd7c2dc#diff-31e158b2a76969d02f7797718c42375fd5b742e80684080b4fccaab00a8323b9R17-R22), [here](https://github.com/PayvoHQ/wallet/commit/d857fd43c91c9a069be960e23145e72dddef6390#diff-46a37ff2a309c900cc5a3847a4363def27879ee9d3e80290914b9a48cac710f6R119), ...

**Review by samharperpittam — APPROVED** [review]

**Pull request merged**